package.json

```diff
@@ -19,9 +18,8 @@
   "scripts": {
     "test": "jest",
     "test-update-snapshot": "jest -u",
-    "cp-css": "./scripts/copy-css.sh",
```

Comment:
I cleaned up the rollup config so that this is no longer necessary. Rollup now outputs everything into `dist`.